extra-webpack.setup.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  resolve: {
+    fallback: {
+      url: false,
+    },
+  },
+  node: {
+    global: true,
+  },
+};
```

Comment:
Instead of creating a new extra-webpack.config.js, that is used for all test runs,
can put fallback `url: false` only in [`karma.conf.js` of the setup lib](https://github.com/SAP/spartacus/blob/develop/core-libs/setup/karma.conf.js#L46-L49)?